pos_partner_deselection/static/description/index.html

```diff
@@ -0,0 +1,94 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan" style="color:#875A7B;">Partner deselection</h2>
+            <h3 class="oe_slogan">POS Partner deselection in opened POS on expiry the predefined time interval after the customer was set.</h3>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container">
+  <div class="oe_row oe_spaced">
+    <div class="oe_span12">
+      <p class="oe_mt32">
+	<div class="alert alert-warning" style="padding:0.3em 0.6em; font-size: 150%;">
+	 It's a good solution to eliminate human error, avoid fraud in case of customers are changing their choice. </br>
+This module automatically allows deselecting customers after some amount of time.
+	</div>
+      </p>
+    </div>
+  </div>
+</section>
+
+  <section class="oe_container oe_dark">
+  <div class="oe_row oe_spaced">
+    <h3 class="oe_slogan">How It Works</h3>
+    <div class="oe_span12 text-center">
+      <p class="oe_mt32">
+ 	<font style="font-size: 120%;">
+	 Administrator can set up the deselection interval and select the Customer.	    
```

Comment:
It's Administrator (or more precisely user with level access "POS Manager") who can set up, but it's not necessarily the same user who will use it. Anyway, configuration and selecting a customer is a different process in different pages. It doesn't look good for me to say this at the same sentence

Reply:
updated